client/flutter/lib/components/home_page_sections/home_page_header.dart

```diff
@@ -1,172 +1,195 @@
+                            variant: TypographyVariant.button,
+                            style: TextStyle(color: Constants.primaryColor),
+                            textAlign: TextAlign.center,
+                          ),
+                        ),
+                        onPressed: () {
+                          return this.link.open(context);
+                        },
                       ),
                     ),
                   ],
                 ),
-              ),
-              SizedBox(
-                height: 32,
-              ),
-              ThemedText(
-                this.title,
-                variant: TypographyVariant.title,
-                style: TextStyle(
-                  color: CupertinoColors.white,
-                ),
-                textAlign: TextAlign.left,
-              ),
-              ThemedText(
-                this.subtitle,
-                variant: TypographyVariant.body,
-                style: TextStyle(
-                  color: CupertinoColors.white,
-                ),
-              ),
-              Container(
-                height: 24,
-              ),
-              Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: [
-                  this.svgAssetName != null
-                      ? SvgPicture.asset(this.svgAssetName)
-                      : Container(),
-                  Padding(
-                    padding: EdgeInsets.symmetric(
-                      horizontal: 12,
-                    ),
-                    child: CupertinoButton(
-                      borderRadius: BorderRadius.circular(50),
-                      padding: EdgeInsets.symmetric(
-                        horizontal: 32,
-                        vertical: 8,
-                      ),
-                      color: CupertinoColors.white,
-                      child: ThemedText(
-                        this.buttonText,
-                        variant: TypographyVariant.button,
-                        style: TextStyle(color: Constants.primaryColor),
-                      ),
-                      onPressed: () {
-                        return this.link.open(context);
-                      },
-                    ),
-                  ),
-                ],
-              ),
-              if (this.svgAssetName == null)
-                Container(
-                  height: 40,
-                ),
+              )
             ],
           ),
-        ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildLogoSection() {
+    return Container(
+      constraints: BoxConstraints(minHeight: 48.0),
+      padding: EdgeInsets.symmetric(
+        horizontal: 24.0,
+      ),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          Container(
+            color: CupertinoColors.white,
+            height: 48.0,
```

Comment:
The 48's used here and below - should they based on scale factor?

Reply:
Good call - I'd left this from the previous implementation, but I will update